cyvcf2/cli.py

```diff
@@ -0,0 +1,159 @@
+import logging
+from datetime import datetime
+
+import coloredlogs
+import click
+
+from . import VCF
+
+from . import __version__
+
+log = logging.getLogger(__name__)
+LOG_LEVELS = ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']
+
+
+def print_header(vcf_obj, include, exclude, silent=False):
+    """Print the header of a vcf obj
+    
+    Parameters
+    ----------
+    vcf_obj: cyvcf2.VCF
+    include: tuple
+        set of strings with info fields that should be included
+    exclude: tuple
+        set of strings with info fields that should be excluded
+    silent: bool
+        If stuff should be printed or not
+    """
+    for header_line in vcf_obj.raw_header.split('\n'):
+        if len(header_line)>0:
+            print_line = True
+            if include:
+                if 'INFO' in header_line:
+                    print_line = False
+                    for inclusion in include:
+                        if inclusion in header_line:
```

Comment:
I agree on these, it was a bit sloppy of me, corrected print_header according to your suggestions. Found it a bit hard to do anything about print_variant though...